tests/ut/codegen/test_if_tensor_return_partition_view.py

```
@@ -0,0 +1,154 @@
+def test_no_tensor_view_scf_if_result(scratch_mlir: str):
+    """Tensors must NOT be routed through scf.if results (issue #1533).
+
+    Routing a TensorType through scf.if retypes it to a fully-dynamic
+    !pto.tensor_view<?x?>, which ptoas rejects under partition_view. Tensors are
+    in-place mutable references, so the merged value is bound to the shared
+    backing tensor_view instead of an scf.if result.
+    """
+    offending = [line for line in _scf_if_header_lines(scratch_mlir) if "tensor_view" in line]
+    assert not offending, (
+        "scf.if must not yield !pto.tensor_view results for in-place tensors; "
+        f"found:\n" + "\n".join(offending) + f"\n\nfull MLIR:\n{scratch_mlir}"
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)